docs/version0.27.md

```diff
@@ -27,15 +27,19 @@
 The following new features and notable changes since v 0.26.0 are included in this release:
 
 - [New `-XX:[+|-]AdaptiveGCThreading` option added](#new-xx-adaptivegcthreading-option-added)
+- [Improved time zone information added to a Java dump file](#improved-time-zone-information-added-to-a-java-dump-file)
 
 ## Features and changes
 
-### Performance improvements
 
 ### New `-XX:[+|-]AdaptiveGCThreading` option added
 
 To optimize performance, you can now enable adaptive threading to automatically tune the number of active parallel garbage collection (GC) threads. When enabled, the GC thread count is dynamically adjusted from collection cycle to cycle to account for changes in the workload and consider the available resources. When parallel workloads decrease, less threads can be used, reducing the overhead and freeing up resources for other processing activities.
 
 Use the [`-xgcmaxthreads`](xgcmaxthreads.md) option with the [`-XX:+AdaptiveGCThreading`](xxadaptivegcthreading.md) option to specify a thread count limit.
 
+### Improved time zone information added to a Java dump file
```

Comment:
I would say '... added to Java dump files' (it's added to all such files, not just one).

Is this the target of the reference on line 30? If so, that reference needs to be updated accordingly.

Reply:
Updated as suggested (in all relevant places)